specification/validated-relationships-service-api.yaml

```diff
@@ -317,24 +319,44 @@ components:
                   type: string
                   description: The patient's NHS number.
                   example: "9000000001"
+        subject: 
+          type: object
+          description: The person who is the subject.
+          properties:
+            type:
+              type: string
+              enum: [Patient] # Here: https://simplifier.net/packages/hl7.fhir.r4.core/4.0.1/files/81189/~details this looks like it should be a URI
+            identifier:
+              description: An identifier - identifies some entity uniquely and unambiguously. Typically this is used for business identifiers.
+              properties:
+                system:
+                  type: string
+                  description: Codesystem URL for the subject's NHS number.
+                  enum: ["https://fhir.nhs.uk/Id/nhs-number"]
+                value:
+                  type: string
+                  description: The subject's NHS number.
+                  example: "9000000002"
         questionnaire:
           type: string
           description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+          example: "Questionnaire/01dc6813-3421-4d14-948d-a4888241add1"
         item:
           type: array
           description: A group or question item from the original questionnaire for which answers are provided.
           items:
             oneOf:
-              - $ref: '#/components/schemas/QuestionnaireItem_ProxyDetails'
-              - $ref: '#/components/schemas/QuestionnaireItem_PatientDetails'
-              - $ref: '#/components/schemas/QuestionnaireItem_RequestedServices'
+              - $ref: '#/components/schemas/QuestionnaireItem_RelatedPerson'
+              - $ref: '#/components/schemas/QuestionnaireItem_ParentalApplicationSupplementaryDetails'
+              - $ref: '#/components/schemas/QuestionnaireItem_Patient'
+              - $ref: '#/components/schemas/QuestionnaireItem_RequestedAccess'
 
-    QuestionnaireItem_ProxyDetails:
+    QuestionnaireItem_RelatedPerson:
       type: object
       properties:
         linkId:
           type: string
-          enum: ["proxy_details"]
+          enum: ["relatedPerson"]
         text:
```

Comment:
Suggest adding an example "Details about the proxy" for parity with most other fields, currently this one comes out as `"text": "string"` in the generated swagger docs.